### ISSUE TITLE
set pieceIcons throwing errors

### ISSUE THREAD

**Issue by miradnan** [grzegorz103/ngx-chess-board#121]
I am trying to set pieceIcon with a icon theme, but it doesn't work. The console throws the following error.

```
ERROR TypeError: Cannot set property 'pieceIconInput' of undefined
    at NgxChessBoardComponent.set pieceIcons [as pieceIcons] (ngx-chess-board.js:2444)
    at Object.ngOnChangesSetInput [as setInput] (core.js:1517)
    at setInputsForProperty (core.js:10901)
    at elementPropertyInternal (core.js:9948)
    at ɵɵproperty (core.js:14700)

online ngx-chess-board.js:2444

set pieceIcons(pieceIcons) {
        this.pieceIconManager.pieceIconInput = pieceIcons;
}
```
Any examples on how a piece theme can be set?

**Comment by grzegorz103** [comment]
Hi, which version do you got? I fixed this bug in v2.1.2. 

Here's example how I set:

In html:

   <ngx-chess-board #board [size]="size"
                     [pieceIcons]="icons"
    ></ngx-chess-board>

In ts file:

    icons: PieceIconInput = {
        blackBishopUrl: '/assets/bishop-black.svg',
        blackKingUrl: '', // Same as above
        blackKnightUrl: '',
        blackPawnUrl: '',
        blackQueenUrl: '',
        blackRookUrl: '',
        whiteBishopUrl: '',
        whiteKingUrl: '',
        whiteKnightUrl: '',
        whitePawnUrl: '',
        whiteQueenUrl: '',
        whiteRookUrl: ''
    };

**Comment by miradnan** [comment]
Works in this version. I had 2.0.9 and it didn't work. Thanks for the quick response.
